system/Database/DataConverter/Cast/ArrayCast.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Database\DataConverter\Cast;
+
+/**
+ * Class ArrayCast
+ *
+ * PHP: array <--> DB column: string
+ *
+ * @extends BaseCast<mixed[], string, mixed>
+ */
+class ArrayCast extends BaseCast implements CastInterface
+{
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
Docblocks are always inherited, these are unnecessary. Also the inline version `{}` shouldn't be used standalone.

Reply:
Okay, removed.